### PR TITLE
Fixed bugs and card view styles

### PR DESCRIPTION
**Pull request by MechanicKim**
I will write a history on each codes.

Simply put,
1. After sign out, when user sign in again, we should not come to general information page. So, I changed this flow.
2. About card view layout, I talked to @Stacyyya and fixed some styles. But, It is not complete yet.

### CODE REVIEW

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -24,6 +24,9 @@ class Login extends Component {
       loaded: false,
       tokenValid: false,
     };
+  }
+
+  componentDidMount() {
```
Comment (MechanicKim) [inline src/components/Login/Login.js:29]: Sometimes, hasToken method makes a problem. So, I moved this method into in componentDidMound method.

File: src/components/Login/Login.js [hunk 2]
```diff
@@ -48,6 +51,8 @@ class Login extends Component {
       } else {
         Actions.generalInfo({ me: profile });
       }
+    } else {
```
Comment (MechanicKim) [inline src/components/Login/Login.js:54]: Now, it is possible situation which is that profile and error are all null.
So, I added else statement.

File: src/components/Login/Login.js [hunk 3]
```diff
@@ -179,7 +184,11 @@ class Login extends Component {
     if (error) {
       this.alert('Sever error(Profile)! Please sign in again.');
     } else if (profile) {
-      Actions.generalInfo({ me: profile });
+      if (profile.status === 200) {
```
Comment (MechanicKim) [inline src/components/Login/Login.js:187]: To solve sequential problem, I added it
@SarahhhC said that code 200 is for old user and 201 is for 201.

File: src/components/UserList/UserList.js [hunk 4]
```diff
@@ -44,6 +44,7 @@ class UserList extends Component {
         }),
       });
 
+      result[result.length - 1].last = true;
```
Comment (SarahhhC) [inline src/components/UserList/UserList.js:47]: What is this code for?

File: ios/Yoda/AppDelegate.m [hunk 5]
```diff
@@ -86,7 +86,7 @@ - (void)userNotificationCenter:(UNUserNotificationCenter *)center didReceiveNoti
 
 //You can skip this method if you don't want to use local notification
 -(void)application:(UIApplication *)application didReceiveLocalNotification:(UILocalNotification *)notification {
-  [[NSNotificationCenter defaultCenter] postNotificationName:FCMNotificationReceived object:self  userInfo:notification.userInfo];
+  [[NSNotificationCenter defaultCenter] postNotificationName:FCMNotificationReceived object:self + userInfo:notification.userInfo];;
```
Comment (SarahhhC) [inline ios/Yoda/AppDelegate.m]: delete ;

File: src/components/UserList/UserList.js (hunk 4 above)
Comment (MechanicKim) [reply src/components/UserList/UserList.js:47]: To give margin right to last card, I wrote this code.
You can find a code giving margin in Row.js

**Comment by SarahhhC** [comment]
Thanks @MechanicKim Nice work! 👍 LGTM

**Comment by SarahhhC** [comment]
LGTM 🌵

**Pull request merged**